spark-doris-connector/src/main/java/org/apache/doris/spark/load/DorisStreamLoad.java

```diff
@@ -222,13 +226,34 @@ public long load(Iterator<InternalRow> rows, StructType schema)
             String loadUrlStr = String.format(loadUrlPattern, getBackend(), db, tbl);
             this.loadUrlStr = loadUrlStr;
             HttpPut httpPut = getHttpPut(label, loadUrlStr, enable2PC, schema);
-            RecordBatchInputStream recodeBatchInputStream = new RecordBatchInputStream(RecordBatch.newBuilder(rows)
-                    .format(dataFormat)
-                    .sep(FIELD_DELIMITER)
-                    .delim(LINE_DELIMITER)
-                    .schema(schema)
-                    .addDoubleQuotes(addDoubleQuotes).build(), streamingPassthrough);
-            httpPut.setEntity(new InputStreamEntity(recodeBatchInputStream));
+
+            if(StringUtils.isNotEmpty(compressType)){
+                if("gz".equals(compressType.toLowerCase())){
+                    if(dataFormat.equals(DataFormat.CSV)){
```

Comment:
Is it possible to judge directly this way?  `if("gz".equals(compressType.toLowerCase()) && dataFormat.equals(DataFormat.CSV) )`

Reply:
No problem。I have made an update。